### DISCUSSION TITLE
How to make sure file managers action run from the directory they're called from and not the home folder?

### DISCUSSION THREAD

**Discussion by Rabcor** [lxqt/lxqt#2577] [Q&A] [answered]
I have an issue where in pcmanfm-qt if I run a filemanager action by right clicking, it executes from the home directory instead of the directory I'm actually in, is there any easy way to fix this or is it maybe a bug?

**Comment by tsujan** [comment]
[Profile Definition](https://github.com/lxqt/pcmanfm-qt/wiki/custom_actions#profile-definition) → Exec → List of parameters

**Reply by Rabcor** [reply]
I'm not seeing anything that'd ensure it runs from the folder it's executed from instead of the home directory though.

**Comment by Rabcor** [comment, marked as answer]
Just in case anyone is actually looking for how to do this, the answer is to use

`Exec=sh -c 'cd %d && command'`
